src/SentryLink.ts

```diff
@@ -62,6 +63,17 @@ export class SentryLink extends ApolloLink {
               // We must have a breadcrumb if attachBreadcrumbs was set
               (breadcrumb as GraphQLBreadcrumb).data.fetchResult = result;
             }
+
+            if (
+              this.options.attachBreadcrumbs.includeError &&
+              result.errors &&
+              result.errors.length > 1
```

Comment:
Why is the check `result.errors.length > 1`? shouldn't this be true for all gql requests that include at least one error but are otherwise not server errors?

Reply:
Good catch, this should definitely by `> 0`.